### PR TITLE
541 encapsulate db name

### PR DESCRIPTION
**Pull request by index-git**
Part of issue #541

- [x] Tests
- ~~Layman Test Client~~
- [ ] Changlelog
- [ ] Documentation

Optionally
- Add dependency to doc/dependencies.md

### CODE REVIEW

File: src/layman/layer/db/__init__.py [hunk 1]
```diff
@@ -135,7 +149,7 @@ def check_new_layername(workspace, layername, conn_cur=None):
         cur.execute(f"""SELECT n.nspname AS schemaname, c.relname, c.relkind
     FROM   pg_class c
     JOIN   pg_namespace n ON n.oid = c.relnamespace
-    WHERE  n.nspname IN ('{workspace}', '{settings.PG_POSTGIS_SCHEMA}') AND c.relname='{layername}'""")
+    WHERE  n.nspname IN ('{workspace}', '{settings.PG_POSTGIS_SCHEMA}') AND c.relname='{table_name}'""")
     except BaseException as exc:
         logger.error(f'check_new_layername ERROR')
         raise LaymanError(7) from exc
```
Comment (jirik) [inline src/layman/layer/db/__init__.py:155]: LaymanError 7 is maybe not needed any more
LaymanError 9 is maybe not needed any more

File: src/layman/layer/db/__init__.py [hunk 2]
```diff
@@ -145,14 +159,15 @@ def check_new_layername(workspace, layername, conn_cur=None):
 
 
 def get_text_column_names(workspace, layername, conn_cur=None):
```
Comment (jirik) [inline src/layman/layer/db/__init__.py:161]: Use table_name as optional parameter for this other methods in this file (if reasonable)

File: src/layman/layer/db/table.py [hunk 3]
```diff
@@ -53,6 +57,7 @@ def delete_layer(workspace, layername, conn_cur=None):
 
 
 def set_layer_srid(workspace, layername, srid, *, conn_cur=None):
```
Comment (jirik) [inline src/layman/layer/db/table.py:59]: Add table_name as optional parameter

**Review by jirik — CHANGES_REQUESTED** [review]

**Comment by index-git** [comment]
Continue in https://github.com/LayerManager/layman/pull/624

**Pull request closed without merging**